### PR TITLE
property should be stored as hash in Adjudicator Contract

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: contracts/UniversalAdjudicationContract.sol [hunk 1]
```diff
@@ -90,45 +93,47 @@ contract UniversalAdjudicationContract {
...
      * @dev Decide the game decision with given witness
      */
-    function decideClaimWithWitness(bytes32 _gameId, bytes[] memory _witness)
-        public
-    {
-        types.ChallengeGame storage game = instantiatedGames[_gameId];
+    function decideClaimWithWitness(
+        types.Property memory _property,
+        bytes[] memory _witness
+    ) public {
```
Comment (tkmct) [inline contracts/UniversalAdjudicationContract.sol:105]: what about adding modifier to check if a challenge game for given propertyHash is instantiated?

Comment (syuhei176) [reply contracts/UniversalAdjudicationContract.sol]: must be added!
I fixed that and added test case "can't decide not initiated game".

File: contracts/UniversalAdjudicationContract.sol [hunk 2]
```diff
@@ -26,6 +26,15 @@ contract UniversalAdjudicationContract {
     event ClaimDecided(bytes32 gameId, bool decision);
     event ChallengeRemoved(bytes32 gameId, bytes32 challengeGameId);
 
+    modifier isInitiated(types.Property memory property) {
```
Comment (tkmct) [inline contracts/UniversalAdjudicationContract.sol:29]: Great!

**Comment by syuhei176** [comment]
This is corresponding PR.
https://github.com/cryptoeconomicslab/gazelle/pull/307

**Pull request merged**